website/pages/docs/glossary/_meta.json

```
@@ -0,0 +1,9 @@
+{
+  "toc": "ToC",
+  "what-is-the-modern-data-stack": "What is the Modern Data Stack?",
+  "what-is-infrastructure-data-lake": "What is Infrastructure Data Lake?",
+  "what-is-cspm": "What is a CSPM?",
+  "what-is-ciem": "What is a CIEM?",
+  "what-is-asm": "What is ASM (Attack Surface Management)?",
```

```suggestion
  "what-is-asm": "What is ASM?",
```

Either we use acronyms for all and unwrap them in parentheses, or only use acronyms, but we should be consistent

---

I think we want to have both due to SEO